engines/grim/gfx_opengl.cpp

```diff
@@ -1957,33 +1957,12 @@ void GfxOpenGL::drawRectangle(const PrimitiveObject *primitive) {
 		glVertex2f(x1, y2 + 1);
 		glEnd();
 	} else {
-		glBegin(GL_QUADS);
-
-		// top line
+		glLineWidth(_scaleW);
```

Comment:
I'd say it's ok to keep it the way you did if other methods do the same approximation.
To my understanding, _scaleW and _scaleH are not equal to one when the screen size is not the same as the game size which is not possible currently (but would be very nice). And even then, _scaleW and _scaleH would have different values only when the aspect ratio correction is disabled.